modules/presets/collection.js

```diff
@@ -93,81 +115,102 @@ export function presetCollection(collection) {
             });
 
             // matches value to preset.name
-            var leading_name = searchable
-                .filter(function(a) {
-                    return leading(a.name().toLowerCase());
-                }).sort(sortNames);
+            var leading_name = take(searchable, function(a) {
+                if (leading(a.name().toLowerCase())) return { preset: a };
+            });
+
+            // matches value to preset.aliases values
+            var leading_aliases = take(searchable, function(a) {
+                var aliases = a.aliases();
+                for (var i = 0; i < aliases.length; i++) {
+                    var alias = aliases[i];
+                    if (leading(alias.toLowerCase())) return { preset: a, alias: alias };
+                }
+            });
 
             // matches value to preset.terms values
-            var leading_terms = searchable
-                .filter(function(a) {
-                    return (a.terms() || []).some(leading);
-                });
+            var leading_terms = take(searchable, function(a) {
+                var terms = a.terms();
+                for (var i = 0; i < terms.length; i++) {
+                    var term = terms[i];
+                    if (leading(term)) return { preset: a, term: term };
+                }
+            });
 
             // matches value to preset.tags values
-            var leading_tag_values = searchable
-                .filter(function(a) {
-                    return Object.values(a.tags || {})
-                        .filter(function(val) { return val !== '*'; })
-                        .some(leading);
-                });
+            var leading_tag_values = take(searchable, function(a) {
+                var tagValues = Object.values(a.tags || []);
+                for (var i = 0; i < tagValues.length; i++) {
+                    var tagValue = tagValues[i];
+                    if (tagValue !== '*' && leading(tagValue)) {
+                        return { preset: a, tagValue: tagValue };
+                    }
+                }
+            });
```

Comment:
Both `tagValue` and `term` could as well be just booleans `isTagMatch`, `isTermMatch` or similar or as well be left out completely because it is currently not used in the UI besides sorting (which did not happen before). It is simply "free" to record this info at this point.